backend-old/ohq/settings.py

```diff
@@ -91,9 +91,11 @@
 # Database
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 
-DATABASES = {}
-DATABASES['default'] = dj_database_url.config(conn_max_age=600, default=config('DATABASE_URL'))
-
+DATABASES = {
+    "default": dj_database_url.config(
+        default="postgres://postgres:postgres@localhost:5432/postgres"
```

Comment:
Is there a reason why the default is PostgreSQL (are there any PostgreSQL features that we use)? Otherwise, it would be better to just have SQLite as the default database for development.

This just simplifies the development environment setup process, since people don't have to figure out how to install and configure a PostgreSQL server. Our other products have SQLite in development and PostgreSQL in production.

Reply:
Ah, I see this is a `backend-old` folder. This may not apply since we're not using this code.